WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationView.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import SwiftUI
+
+struct TapToPayEducationView: View {
+    @StateObject private var viewModel: TapToPayEducationViewModel
+
+    init(viewModel: TapToPayEducationViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 8) {
+                TabView(selection: $viewModel.selectedStep) {
+                    ForEach(0..<viewModel.steps.count, id: \.self) { index in
```

Comment:
Perhaps having an identifiable step view model would avoid accessing the array by index?

Reply:
It would be possible to use viewModel.steps directly but I haven't found a reliable way to tie it all to the  `viewModel.selectedStep` which could be controlled from code as well. Playing with an index to implement next, back, and skip actions is quite handy.